project007/webapp/controller/ListReport.controller.js

```diff
@@ -45,10 +47,14 @@ sap.ui.define(
        */
       onCancelButton: function () {
         var oStateModel = this.getView().getModel("stateModel");
+        var oODataModel = this.getView().getModel("oData");
 
         this.byId("CategoriesTable").getItems().forEach((elem) => (elem.mProperties.selected = false));
```

Comment:
you are not allowed to use that elem.mProperties

use public methods

https://sapui5.hana.ondemand.com/#/api/sap.m.ListBase%23methods/removeSelections